src/loaders/assetLoader.js

```diff
@@ -137,9 +140,9 @@ module.exports = async function assetLoader() {
     );
   }
 
-  const hashes = pairs.map(item => hasha(item.content, { algorithm: 'md5' }));
+  const hashes = pairs.map(item => hasha(item.content, {algorithm: 'md5'}));
 
-  let info: ?{ width: number, height: number, type: string };
```

Comment:
Can we revert this in upstream `callstack` preset? I think spaces in between `{` are already widely used across our internal projects 💯